maven-parent/pom.xml

```diff
@@ -573,6 +597,15 @@
                       ${project.build.directory}/bundled-docker-build-tmp/distro</contextDir>
                   </build>
                 </image>
+                <image>
+                  <name>${docker.push.registry.username}/%a-senaite-sso</name>
+                  <build>
+                    <dockerFile>
+                      ${project.build.directory}/bundled-docker-build-tmp/bundled-docker/senaite/Dockerfile-sso</dockerFile>
```

Comment:
In the end, we can't reuse the same base image w/o SSO. The build failure is because the `Dockerfile-sso` isn't available. It is provided by https://github.com/ozone-his/ozone-docker-compose/pull/135